src/HL/View/Home/Features.hs

```diff
@@ -80,8 +80,10 @@ statically =
 concurrent :: Html ()
 concurrent =
   do h2_ "Concurrent"
-     p_ "Haskell lends itself well to concurrent programming due to its explicit \
-       \handling of effects. Its flagship compiler, GHC, comes with a high-\
+     p_ "Effects such as IO and mutation are made explicit in Haskell\
```

Comment:
Quite probably, I wasn't able to test the changes before sending the pull request and probably missed that one because it was the first thing I changed. I can make the change and resubmit if you like.